File: 백준/Bronze/34934. 신규 학과/신규 학과.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int yr,n = scan.nextInt();
        String sub;
        for(int i=0;i<n;i++) {
            sub=scan.next();
            yr=scan.nextInt();
            if(yr==2026) {
                System.out.print(sub);
                break;
            }
        }
    }
}

File: 백준/Bronze/34934. 신규 학과/신규 학과.cc
#include <iostream>
using namespace std;
int main(void) {
    int n,yr;
    string sub;
    cin>>n;
    for(int i=0;i<n;i++) {
        cin>>sub>>yr;
        if(yr==2026) {
            cout<<sub;
            break;
        }
    }
    return 0;
}